processes/refresh.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+while IFS= read -r -d '' file
+do
+  echo "Rendering graph ${file} into ${file}.png"
+  dot -Tpng -o "$file.png" "$file"
```

Comment:
To avoid `blah.dot.png`:
```
...
  png="${file%.dot}.png"
  echo "Rendering graph ${file} into ${png}"
...
```

Reply:
The `.dot.png` is preexisting so I'm reluctant to change that as well in this PR - could that happen as a follow up maybe?

Reply:
I suppose 😞